data-prepper-plugins/mutate-event-processors/src/main/java/org/opensearch/dataprepper/plugins/processor/mutateevent/TargetType.java

```diff
@@ -18,30 +19,34 @@
 import java.util.stream.Collectors;
 
 public enum TargetType {
-    INTEGER("integer", new IntegerConverter()),
-    STRING("string", new StringConverter()),
-    DOUBLE("double", new DoubleConverter()),
-    BOOLEAN("boolean", new BooleanConverter()),
-    LONG("long", new LongConverter());
+    INTEGER(Type.INTEGER, new IntegerConverter()),
```

Comment:
Not sure if there are any tests for the config but just want to verify that we aren't breaking the config and requiring caps to not break backwards compatibility with `integer` instead of `INTEGER`

Reply:
I added that here:

https://github.com/opensearch-project/data-prepper/pull/4496/files#diff-8ba2f687e9a9672064f347a04a203fc3b99149fe0b764c367c82b8e0bb303d67R34-R45